components/expression/expression-renderer.js

```diff
@@ -22,13 +22,43 @@ genotet.ExpressionRenderer = function(container, data) {
   this.geneLabelWidth_ = 0;
 
   /**
-   * The maximum height the vertical experiment condition labels.
+   * The factor of the maximum width of the horizontal gene labels.
+   * @private {number}
+   */
+  this.geneLabelWidthFactor_ = 8.725;
+
+  /**
+   * The maximum height of the vertical experiment condition labels.
    * This value will be zero when condition labels are not shown.
    * @private {number}
    */
   this.conditionLabelHeight_ = 0;
 
   /**
+   * The factor of the maximum height of the vertical experiment condition labels.
+   * @private {number}
+   */
+  this.conditionLabelHeightFactor_ = 6.501175;
```

Comment:
If this is a constant, then it should be named as CONDITION_LABEL_HEIGHT_FACTOR_ and be an attribute of the class prototype.